src/input-screen/index.js

```diff
@@ -83,43 +87,44 @@ class FileSelection extends PureComponent {
 
 		return (
 			<Fragment>
-				<DropZoneTarget>
-					<h2>Import WordPress</h2>
-					<div>Howdy! Upload your WordPress eXtended RSS (WXT) file and we'll import the posts, pages, comments, custom fields, categories, and tags into this site.</div>
-					<div>Choose a WXR (.xml) file to upload, or drop a file here, and your import will begin</div>
-					{ ! url && (
+				<h2>Import WordPress</h2>
+				<div>Howdy! Upload your WordPress eXtended RSS (WXT) file and we'll import the posts, pages, comments, custom fields, categories, and tags into this site.</div>
```

Comment:
awesome how the syntax highlighting breaks on text nodes 😠

Reply:
Very 👎 :(